### DISCUSSION TITLE
Some issues reports

### DISCUSSION THREAD

**Discussion by DamienGeiger** [keiwando/toonsquid#271] [Ideas]
Hi !
It looks you are working hard on your app, so here some feature reports.
I'm working with vectors only. There is an issue with the stroke width: if I set 10, it won't be the same width depending which shape I am working on (line, square, circle,...)
Plus, I am willing to copy-paste an entire animated vector so that I can have for example two same bees flying. I kind of can do it with the transform layer, but it's not really intuitive. It would be great if we could do it simply (that the position of the control points are dependant of the center of the shape for example and not a fixed position on the grid).
Ah yeah, a grid with magnetism would be amazing !

**Comment by keiwando** [comment]
Hi!
Thanks for the feedback. The stroke width difference comes from the fact that the path tool edits the layer's scale when you touch and drag to create a rectangle or ellipse, but I agree that it feels confusing, so I've noted down to change this in a future update.

Could you maybe clarify what exactly you mean with the "two bees" scenario? If you want to animate a bee once and then use this same animation multiple times, you can use [symbols](https://toonsquid.com/handbook/symbols/animation_clips/) for that. If you want to animate the transform of multiple layers in the exact same way, the transform layer is indeed the way to do it. 
If you [enable pivot editing](https://toonsquid.com/handbook/transform/transform_properties/#editing-pivot) and drag the pivot of the transform layer to the center of one of your layers, it also moves the scale and rotation handles of the transform layer there, if I understood your issue with that correctly.
